dataprofiler/data_readers/csv_data.py

```diff
@@ -106,6 +106,10 @@ def quotechar(self):
     def header(self):
         return self._header
 
+    @property
+    def is_structured(self):
```

Comment:
let's add a comment describing the property. I know others may not, but we should add those in another task and make sure to add in future.